src/mqtt.js

```diff
@@ -608,6 +620,31 @@ function convertHexIdToNumericId(hexId) {
     return BigInt('0x' + hexId.replaceAll("!", ""));
 }
 
+/**
+ * Obfuscates overly precise location data
+ * @param coordinateI the original coordinate (lat or lng) as an integer
+ * @returns {coordinateI} the obfuscated coordinate as an integer
+ */
+function obfuscateCoordinate(coordinateI) {
+    // Only obfuscate if option enabled
+    if (!obfuscatePositions) return coordinateI;
+
+		// Coordinates come in as integers
+    const coordinateS = (coordinateI * .0000001).toString();
```

Comment:
@liamcottle Need you to validate that this approach works. I am not familiar with how the coordinates are encoded into an integer. Is it just a straight `* 10000000`?